src/pkg/podman/podman.go

```diff
@@ -332,3 +333,83 @@ func SystemMigrate(ociRuntimeRequired string) error {
 
 	return nil
 }
+
+// internalError serves for representing errors printed by Podman to stderr
+type internalError struct {
```

Comment:
It seems like the `internalError` implementation is the kind of thing that should live in a different file. It's self-contained and somewhat separate from the rest of `podman.go`, which is mostly about wrapping `podman` invocations.

Reply:
Sure, sounds good!